Core/src/Surfaces/detail/AnnulusBoundsHelper.cpp

```diff
@@ -0,0 +1,65 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Surfaces/detail/AnnulusBoundsHelper.hpp"
+
+#include "Acts/Surfaces/AnnulusBounds.hpp"
+#include "Acts/Utilities/VectorHelpers.hpp"
+
+#include <iostream>
+
+std::tuple<std::shared_ptr<Acts::AnnulusBounds>, Acts::Transform3>
+Acts::detail::AnnulusBoundsHelper::create(const Transform3& transform,
```

Comment:
Can't this be a constructor of annulus bounds? That's how I added the construction from the cartesian defining parameters over in Athena, maybe that would also be a good fit here?

Reply:
Can you give me details about this? Since I followed up on this branch and want to merge parts of it in #3256?